crates/uv/tests/pip_install.rs

```diff
@@ -1925,95 +1925,51 @@ fn install_only_binary_all_and_no_binary_all() {
               anyio>=3.0.0,<=3.6.2
               anyio>=3.7.0,<=3.7.1
               anyio>=4.0.0
-          and anyio==1.0.0 has no usable wheels and building from source is disabled, we can conclude that any of:
+          and any of:
+              anyio==1.0.0
+              anyio==1.1.0
+              anyio==1.2.0
+              anyio==1.2.1
+              anyio==1.2.2
```

Comment:
Now we enumerate all the versions in a single clause instead of having a clause for each! I think now we can simplify this range with the known available versions and it'll be way better.

Reply:
https://github.com/astral-sh/uv/pull/6155